hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/PravegaOutputFormat.java

```diff
@@ -63,6 +63,8 @@
     public static final String SCOPE_NAME = "pravega.scope";
     // Pravega stream name
     public static final String OUT_STREAM_NAME = "pravega.out.stream";
+    // Pravega stream segments
```

Comment:
Why are we changing anything in the `wordcount` example here? If we have classes shared across examples, then we need to organize them better, this is messy.

Reply:
Moved the outputformat, recordwriter to common place.

Reply:
I understand why we have custom `PravegaOutputFormat` implementation in samples since it was not part of the Hadoop connectors before. Now that we have `PravegaOutputFormat` available as part of the hadoop connector (https://github.com/pravega/hadoop-connectors/blob/master/src/main/java/io/pravega/connectors/hadoop/PravegaOutputFormat.java), I think it is misleading if we don't use it in the samples. If we have a need to use a custom implementation, it is worth documenting at least since the samples are supposed to be a reference application on how to make use of the connector?

Reply:
Class name changed to avoid the confusion.
(Separate issue/task needed to update sample code with the latest hadoop connector)